instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/jaeger/JaegerExporterProperties.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.exporters.jaeger;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "opentelemetry.exporter.jaeger")
+public class JaegerExporterProperties {
+
+  private String serviceName = "otel-spring-boot-jaeger-exporter";
+  private String endpoint = "localhost:14250";
```

Comment:
[Suggestion] Create a constant for this port and name it (e.g., default JAEGER port)

Reply:
These fields are not constants. They're default values that will be overwritten by a users configurations. I just set some default values to simplify the setup of this auto configuration package.